pisek/task_jobs/tools.py

```diff
@@ -137,7 +137,7 @@ def _run(self):
 
 
 class PrepareTokenJudge(PrepareJudgeLibJudge):
-    """Compiles the generic token-based judge."""
+    """Compiles the judge-token from judgelib."""
```

Comment:
This should be either "Compiles judge-token", or "Compiles the judge-token judge/program".